docs/integrations/sources/gitlab.md

```diff
@@ -97,7 +94,7 @@ This connector outputs the following streams:
 - [Deployments](https://docs.gitlab.com/ee/api/deployments/index.html)
 - [Group Labels](https://docs.gitlab.com/ee/api/group_labels.html)
 - [Project Labels](https://docs.gitlab.com/ee/api/labels.html)
-- [Epics](https://docs.gitlab.com/ee/api/epics.html) \(only available for GitLab Ultimate and GitLab.com Gold accounts\)
+- [Epics](https://docs.gitlab.com/ee/api/epics.html) \(only available for GitLab Ultimate and GitLab.com Gold accounts. Using iid field as primary key - The internal ID of the epic.\)
```

Comment:
Can we explain how this is more useful than using ID? I think it would be interesting in case we have to do a change related to IID

Reply:
Added description for pk.